packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -171,16 +171,16 @@ class OpPerfTelemetry {
     }
 
     private recordPingTime(latency: number) {
-        this.pongCount++;
         this.pingLatency = latency;
-        // logging one in every 100 pongs
-        if (this.pongCount === 100) {
+        // logging one in every 100 pongs, including the first time.
+        if (this.pongCount % 100 === 0) {
```

Comment:
Not sure how many events we would get, I'd start safe here - with 1000, and if we see we are not getting that many events, I'd reduce it.
We also might want to consider logging it only for "write" clients (DeltaManaget.active === true). This will eliminate case where there are hundreds of idle clients that do not do anything. This will reduce somewhat dataset, but ping latency should not depend on read vs. write connections